[Bug] unable to install

I try to work on #66 but when running `yarn`, I got this:

```sh
yarn
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
warning file-loader@2.0.0: Invalid bin field for "file-loader".
warning mini-css-extract-plugin@0.4.3: Invalid bin field for "mini-css-extract-plugin".
warning terser-webpack-plugin@1.1.0: Invalid bin field for "terser-webpack-plugin".
warning url-loader@1.1.1: Invalid bin field for "url-loader".

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
src\index.js -> dist\index.js
src\react-decorator.js -> dist\react-decorator.js
Successfully compiled 2 files with Babel (3530ms).
Transpiling completed.

npm ERR! could not determine executable to run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\homa\AppData\Local\npm-cache\_logs\2023-04-05T00_12_58_295Z-debug-0.log
```

It seems like `npx typescript` doesn't work.

Maybe adding `typescript` as regular dev deps?

--- a/package.json
+++ b/package.json
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "start": "start-storybook -p 9009 --ci",
-    "prepare": "package-prepare && npx typescript 'src/index.js' --declaration --allowJs --emitDeclarationOnly --outDir dist",
+    "prepare": "package-prepare && yarn build",
+    "build": "tsc \"src/index.js\" --declaration --allowJs --emitDeclarationOnly --outDir dist",
     "build:tests": "rm -rdf .jest-dist && babel ./src -d ./.jest-dist -s",
     "build:readme.md": "jsdoc2md -t docs/readme.hbs -c docs/jsdoc.config.json src/*.js  > README.md; echo",
     "build:readme": "jsdoc2md -t docs/readme.hbs -c docs/jsdoc.config.json src/*.js  > README; echo",
@@ -93,6 +94,7 @@
     "react-dom": "^16.4.0",
     "react-scripts": "2.0.5",
     "react-test-renderer": "^16.4.0",
+    "typescript": "^5.0.3",
     "url-loader": "^2.1.0"
   },
   "peerDependencies": {

